Documentation/Signoffs/Main_Processor.md

```diff
@@ -1,53 +1,79 @@
-## How to submit a signoff or update a signoff:
+# Processing Subsystem
 
-### Submitting Signoff
-1. Create the signoff markdown file 
-2. At the bottom of the page select "create a new branch for this commit and start a pull request"
-3. Submit the changes 
-4. Name the pull request branch "name-signoff-subsystem name"
-5. The comment should be "requesting signoff for name of subsystem"
-6. Submit the pull request
-7. Wait for faculty supervisor feedback on the pull request
+## Function of the Subsystem
 
-### Updating Signoff
+##### Pseudo-Code Flowchart
+<img src="https://user-images.githubusercontent.com/123997954/229912784-1174c224-1c0f-4fb9-8a78-3f503ecb81d3.png" width="60%" height="60%">
 
-1. Edit the signoff markdown file 
-2. Add a description of how you have addressed the feedback from the faculty supervisor
-3. At the bottom of the page select "commit directly to"
-4. Submit the changes 
+The goal of this subsystem is to process the input signal, and output an appropriate response to the ambient noise present in the room. The processing subsystem consists of an ADC converter which will take the input and convert it into digital samples which will be analyzed and processed by the adaptive filter algorithms. The output will then be converted using a DAC converter and outputted to the other parts of the system. The room will be sliced into sections of noise cancelling like the image below where the cyan represents an area of noise control..
 
-Making tables for constraints and BOM will use Excel to Markdown here:
-https://tabletomarkdown.com/convert-spreadsheet-to-markdown/
+<img src="https://github.com/CarsonDPope/Active-Noise-Control-With-Wall-Transmission-Detection/blob/CarsonDPope-patch-1/Documentation/Images/AreaOfEffect.png" width="60%" height="60%">
 
-Example Signoff that was shown in class can be found here:
-https://github.com/Hawk652/Capstone-Guidance-Robot/blob/main/Documentation/Signoffs/Power%20Subsystem.md
+## Constraints
 
-# Required sections
+| No. | Constraints                                                           | Origin            |
+| --- | --------------------------------------------------------------------- | ----------------- |
+| 1   | System shall be powered by standard wall outlets                      | Spec Constraint   |
+| 2   | Shall process and output with a max delay of 1.4 ms*                  | Design Constraint |
+| 3   | Shall have a sample rate of at least 40 kHz                           | Spec Constraint   |
```

Comment:
This isn't something that makes sense to spec. What determines the appropriate sample rate of a system?